ui/js/search.js

```diff
@@ -13,12 +13,54 @@
  * permissions and limitations under the License.
  */
 
-require(['jquery','oae.core', 'jquery.history'], function($, oae) {
+require(['jquery', 'oae.core', 'underscore', 'jquery.history', 'jquery.switchtab'], function($, oae, _) {
 
     // Variable that will be used to keep track of the current
     // infinite scroll instance
     var infinityScroll = false;
 
+    var $switchtab = null;
+    var currSwitchtabId = null;
+
+    /**
+     * Get the search query data in the current state
+     *
+     * @return {Object}     queryData           The query data from the current state
+     * @return {String}     queryData.q         The full-text search query
+     * @return {String[]}   queryData.types     The resource types for which to search
+     * @return {String}     queryData.tenant    The tenant alias in which to search
+     */
+    var _queryData = function() {
```

Comment:
We don't really do `_` for private functions in the UI as pretty much everything is private to the current file/widget